fly/commands/watch.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 type WatchCommand struct {
-	Job   flaghelpers.JobFlag `short:"j" long:"job"   value-name:"PIPELINE/JOB"   description:"Watches builds of the given job"`
-	Build string              `short:"b" long:"build"                               description:"Watches a specific build"`
+	Job   		flaghelpers.JobFlag `short:"j" long:"job"   	value-name:"PIPELINE/JOB"   description:"Watches builds of the given job"`
+	Build 		string              `short:"b" long:"build"                               	description:"Watches a specific build"`
+	Timestamp 	bool				`short:"t" long:"timestamp"								description:"Print with timestamp"`
```

Comment:
The short flag `-t` here will conflict with `-t` for `--target`. I'm having a hard time thinking of a better short flag though.

minor nitpick: `go fmt` on save?

Reply:
How does `-u` sound for UTC or `-z` for timezone or `-i` for the second letter in the word?